legend-pure-m2-functions-pure/src/main/resources/platform_functions/string/ascii.pure

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+native function meta::pure::functions::string::ascii(source:String[1]):Integer[1];
+
+function <<test.Test>> {test.excludePlatform = 'Java compiled'} meta::pure::functions::string::tests::ascii::testSimpleAscii():Boolean[1]
+{
+    assertEquals(0, ascii(''));
+    assertEquals(32, ascii(' '));
+    assertEquals(97, ascii('abc'));
```

Comment:
Is this correct?  What it means to get ascii code of multiple characters?  Or 97 is the ascii code of  'a' and 'bc' are ignored?

Reply:
Yes gets ascii code of first char. Appreciate we should add doc to the function to explain